supplier_management_site/tests/test.py

```diff
@@ -22,3 +32,18 @@ def test_testing_page_contains_correct_translation(self, lang, text):
         request = self.factory.get("/{}/".format(lang))
         response = home(request)
         self.assertContains(response, text)
+
+
+class EmailUtilsTest(TestCase):
+    def tearDown(self):
+        mail.outbox = []
+
+    def test_send_email_notification(self):
+        subject = 'Testing title'
+        message = 'Testing message'
+        recipient_list = ['someone@somemail.com']
+
+        send_email_notification(subject, message, recipient_list)
```

Comment:
Are we actually sending the email on the test? 🤔 It could be good if you mock the send_email function (from django.core.mail import send_mail)